Modules/Loadable/Segmentations/Logic/vtkSlicerSegmentEditorLogic.cxx

```diff
@@ -99,7 +108,7 @@ std::string vtkSlicerSegmentEditorLogic::AddEmptySegment(const std::string& segm
 {
   if (!this->SegmentEditorNode)
   {
-    vtkErrorMacro("" << __func__ << ": Invalid segment editor parameter set node");
+    WarningMacro("Invalid segment editor parameter set node");
```

Comment:
We should not define a macro with such a generic name. For now I suggest to simplify by repeating the `if (this->IsVerbose())`.

Then, in a follow-up pull request, support for conditional logging could be introduced by adding `IsVerbose` to `vtkMRMLAbstractLogic` along with introduction of macros like `vtkMRMLLogicWarningMacro`.